code/harness/c_harness.cpp

```diff
@@ -669,14 +671,30 @@ int main(int argc, char** argv) {
     test_harness.test_check_random_permutations(iterations);
     test_harness.remove_cow_brd();
 
-    test_harness.PrintTestStats(cout);
+    test_harness.PrintTestStats(cout, false);
     cout << endl;
 
     for (unsigned int i = 0; i < Tester::NUM_TIME; ++i) {
       cout << "\t" << (Tester::time_stats) i << ": "
         << test_harness.get_timing_stat((Tester::time_stats) i).count() << " ms"
         << endl;
     }
+    //get the name of the test being run 
+    string path = argv[test_case_idx];
```

Comment:
Could you actually move this to the top of the file. I don't really like the idea of accessing a command line arg so far down in the file :)
Could you also make it `const string` if you don't plan on changing it? Just so it doesn't accidentally get changed later on.